float32[] joint_cmd
---
bool success
